repos/arcade.proj

```diff
@@ -51,5 +51,26 @@
     <RepositoryReference Include="sourcelink" />
   </ItemGroup>
 
+  <Target Name="UsePreviouslySourceBuiltNuGet"
+          Condition="'$(OfflineBuild)' == 'true'"
+          BeforeTargets="RepoBuild">
+    <ItemGroup>
+      <PreviouslySourceBuiltNuGetPackageFile Include="$(PrebuiltSourceBuiltPackagesPath)NuGet.Packaging.*.nupkg" />
+    </ItemGroup>
+
+    <Error
+      Text="Unable to find any previously source built NuGet.Packaging version to use in Arcade SDK tooling."
+      Condition="'@(PreviouslySourceBuiltNuGetPackageFile)' == ''" />
+
+    <ReadNuGetPackageInfos PackagePaths="@(PreviouslySourceBuiltNuGetPackageFile)">
+      <Output TaskParameter="PackageInfoItems" ItemName="PreviouslySourceBuiltNuGetPackageInfo" />
+    </ReadNuGetPackageInfos>
+
+    <PropertyGroup>
+      <PreviouslySourceBuiltNuGetVersion>%(PreviouslySourceBuiltNuGetPackageInfo.PackageVersion)</PreviouslySourceBuiltNuGetVersion>
+      <BuildCommand>$(BuildCommand) /p:NuGetVersion=$(PreviouslySourceBuiltNuGetVersion)</BuildCommand>
```

Comment:
I didn't consider the `PackageVersions.props` stored within the previously source-built dir when I wrote this--I don't think that makes this approach invalid (so we should still merge it for now), but I was running into issues with this in a future prebiult removal and will focus on sharing that infra properly in the future.